tests/framework/PostProcessors/InterfacedPostProcessor/test_historySetSamplingIPP.xml

```diff
@@ -157,10 +156,6 @@
     </MonteCarlo>
   </Samplers>
 
-  <Databases>
-    <HDF5 name="out_db" readMode="overwrite"/>
-  </Databases>
-
```

Comment:
why database is removed?

Reply:
I removed since it was not part of the testing capability